config/config.yaml

```diff
@@ -5,3 +5,8 @@ admincentral_cf_bucket: "bootstrap-awss3cloudformationbucket-19qromfd235z9"
 template_key_prefix: {{ environment_variable.TRAVIS_BRANCH | default("testing") }}
 toil_key_pair: "toil"
 toil_ami: "ami-024a64a6685d05041" #ubuntu 18.04
+stack_tags:
+  Department: "SysBio"
```

Comment:
Neither @wpoehlm nor I are in SysBio, but I also have other questions how this should be attributed; see next question below.

Reply:
Please change Department to "IBC"

Reply:
done